src/SFA.DAS.ApplyService.Web/Controllers/Roatp/GetHelpController.cs

```diff
@@ -0,0 +1,126 @@
+        {
+            var getHelpQuery = new GetHelpWithQuestion();
+
+            if (applicationId.HasValue && applicationId.Value != Guid.Empty)
+            {
+                var sequences = await _qnaApiClient.GetSequences(applicationId.Value);
+                var currentSequence = sequences.Single(x => x.SequenceId == sequenceId);
+                var sections = await _qnaApiClient.GetSections(applicationId.Value, currentSequence.Id);
+                var currentSection = sections.FirstOrDefault(x => x.SectionId == sectionId);
+
+                var page = await _qnaApiClient.GetPage(applicationId.Value, currentSection.Id, pageId);
+
+                getHelpQuery.PageTitle = page.Title;
+                var sequenceConfig = _taskListConfiguration.FirstOrDefault(x => x.Id == sequenceId);
+                if (sequenceConfig != null)
+                {
+                    getHelpQuery.ApplicationSequence = sequenceConfig.Title;
+                }
+                if (currentSection != null)
+                {
+                    getHelpQuery.ApplicationSection = currentSection.Title;
+                }
+
+                var organisationName = await _qnaApiClient.GetAnswerByTag(applicationId.Value, RoatpWorkflowQuestionTags.UkrlpLegalName);
+                if (organisationName != null && !String.IsNullOrWhiteSpace(organisationName.Value))
+                {
+                    getHelpQuery.OrganisationName = organisationName.Value;
+                }
+                else
+                {
+                    getHelpQuery.OrganisationName = "Not available";
+                }
+
+                var organisationUKPRN = await _qnaApiClient.GetAnswerByTag(applicationId.Value, RoatpWorkflowQuestionTags.UKPRN);
+                if (organisationUKPRN != null && !String.IsNullOrWhiteSpace(organisationUKPRN.Value))
+                {
+                    getHelpQuery.UKPRN = organisationUKPRN.Value;
+                }
+                else
+                {
+                    getHelpQuery.UKPRN = "Not available";
+                }
+            }
+            else
+            {
+                // in preamble so we don't have an application set up yet
+                var applicationDetails = _sessionService.Get<ApplicationDetails>(ApplicationDetailsKey);
+                getHelpQuery.PageTitle = title;
+                getHelpQuery.ApplicationSequence = "Preamble";
+                getHelpQuery.ApplicationSection = "Preamble";
+                var ukprn = applicationDetails?.UKPRN.ToString();
+                if (String.IsNullOrWhiteSpace(ukprn))
+                {
+                    ukprn = "Not available";
+                }
+                getHelpQuery.UKPRN = ukprn;
+                var organisationName = applicationDetails?.UkrlpLookupDetails?.ProviderName;
+                if (String.IsNullOrWhiteSpace(organisationName))
+                {
+                    organisationName = "Not available";
+                }
```

Comment:
organisationName = string.IsNullOrWhiteSpace(organisationName) ? "Not available" : organisationName;   

a little less code, but don't care if you leave it